Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+#include <string>
+#include "Person.h"
+using namespace std;
+
+
+
+Person::Person(){
+    age_ = new int(1);
```

Comment:
`age_ = new int();` would work better since you are initializing it to be empty.

Reply:
done